testing/e2e/manifests/clientServer.go

```diff
@@ -21,29 +21,32 @@ var serverContents string
 
 var nonAlphanumericRegex = regexp.MustCompile(`[^a-zA-Z0-9 ]+`)
 
-type testingResources struct {
-	Client  *appsv1.Deployment
-	Server  *appsv1.Deployment
-	Service *corev1.Service
-	Ingress *netv1.Ingress
+type ClientServerResources struct {
```

Comment:
This should still be private. We don't want to expose internal parts of this package externally